### PR TITLE
Docs: clean up getting started

### PR DESCRIPTION
**Pull request by dev2049**
Main change: split quickstart into two, put chat model stuff in a Part 2 doc. Also clean up headers for prettier table of contents

check it out here: https://python.langchain.com/en/dev2049-getting_started_clean

### CODE REVIEW

File: docs/getting_started/getting_started_chat.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
+# Chat Models
+
+This guide walks you through how the core LangChain modules work with chat models. We recommend first reading through the [Quickstart Guide](./getting_started.md), which introduces the core concepts in the context of LLMs.\
+\
+Chat models are a variation on language models. While chat models use language models under the hood, the interface they expose is a bit different: rather than expose a "text in, text out" API, they expose an interface where "chat messages" are the inputs and outputs.\
+\
```
Comment (vowelparrot) [inline docs/getting_started/getting_started_chat.md:6]: Are the backslashes needed if we do double newlines?

File: docs/getting_started/getting_started.md [hunk 2]
```diff
@@ -314,7 +325,10 @@ You can get completions by passing in a single message.
 
 ```python
 chat([HumanMessage(content="Translate this sentence from English to French. I love programming.")])
-# -> AIMessage(content="J'aime programmer.", additional_kwargs={})
+```
+
+```pycon
```
Comment (eyurtsev) [inline docs/getting_started/getting_started.md:330]: :O what's pycon is that for output?!

File: docs/getting_started/getting_started_chat.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+        HumanMessage(content="I love programming.")
+    ],
+    [
+        SystemMessage(content="You are a helpful assistant that translates English to French."),
+        HumanMessage(content="I love artificial intelligence.")
+    ],
+]
+result = chat.generate(batch_messages)
+result
+# -> LLMResult(generations=[[ChatGeneration(text="J'aime programmer.", generation_info=None, message=AIMessage(content="J'aime programmer.", additional_kwargs={}))], [ChatGeneration(text="J'aime l'intelligence artificielle.", generation_info=None, message=AIMessage(content="J'aime l'intelligence artificielle.", additional_kwargs={}))]], llm_output={'token_usage': {'prompt_tokens': 57, 'completion_tokens': 20, 'total_tokens': 77}})
```
Comment (vowelparrot) [inline docs/getting_started/getting_started_chat.md:61]: Could do the ```pycon syntax here right?

**Review by vowelparrot — APPROVED** [review]
I'm a fan

**Comment by eyurtsev** [comment]
This is looking pretty good to me... discussed offline, but let's follow up later with changes to replace Prompt instantiation everywhere with the `from_prompt` classmethod, makes it much easier to use prompts

File: docs/getting_started/getting_started_chat.md (hunk 1 above)
Comment (dev2049) [reply docs/getting_started/getting_started_chat.md:6]: yea sphynx (or our theme or whatever, i don't really understand these things) doesn't give a damn how many new lines i add unfortunately

File: docs/getting_started/getting_started_chat.md (hunk 3 above)
Comment (dev2049) [reply docs/getting_started/getting_started_chat.md:61]: yup, couple places where that's the case, was gonna save for sep pr

Comment (leo-gan) [reply docs/getting_started/getting_started_chat.md:61]: Maybe rename `getting_started.md` to `getting_started_llm.md`, so we have balanced names together with `getting_started_chat.md`? LOGAF low

**Comment by hwchase17** [comment]
mmm seems a bit weird. Basics is installation + LLMs, and then chat is separate? i dont think thats very logical

if anything id prefer to combine more

**Comment by dev2049** [comment]
> mmm seems a bit weird. Basics is installation + LLMs, and then chat is separate? i dont think thats very logical
> 
> if anything id prefer to combine more

part 1 = intro to all modules, mostly in context of LLMs bc those are more established and bit easier to understand
part 2 = go over all modules again but this time in context of chat models, which are the newer abstraction

combining sounds great eventually but think we first need to make the other modules handle either model type more seamlessly

**Pull request closed without merging**